daemon/cmd/state.go

```diff
@@ -455,30 +457,49 @@ func (d *Daemon) initRestore(restoredEndpoints *endpointRestoreState, endpointsR
 
 		go func() {
 			if d.clientset.IsEnabled() {
+				// Configure the controller which removes any leftover Kubernetes
+				// services that may have been deleted while Cilium was not
+				// running. Once this controller succeeds, because it has no
+				// RunInterval specified, it will not run again unless updated
+				// elsewhere. This means that if, for instance, a user manually
+				// adds a service via the CLI into the BPF maps, it will
+				// not be cleaned up by the daemon until it restarts.
+				syncServices := func(localOnly bool) {
+					d.controllers.UpdateController(
+						"sync-lb-maps-with-k8s-services",
+						controller.ControllerParams{
+							Group: syncLBMapsControllerGroup,
```

Comment:
@giorio94 this is breaking the build as it's not defined in v1.14, the original controller down below was just:
```go
			controller.NewManager().UpdateController("sync-lb-maps-with-k8s-services",
					controller.ControllerParams{
						DoFunc: func(ctx context.Context) error {
							return d.svc.SyncWithK8sFinished(d.k8sWatcher.K8sSvcCache.EnsureService)
						},
						Context: d.ctx,
					},
				)
```

can I just skip the group/do I need additional commits?

Reply:
Yes, it is fine to drop the Group field (it only serves observability purposes, and it was not present in v1.14)